docs/source/en/basics/schedule.md

```diff
@@ -0,0 +1,207 @@
+title: Scheduled Tasks
+---
+
+Although the HTTP Server we developed using the framework is a request-response model, there are still many scenarios that need to execute some scheduled tasks, for example:
+
+1. Regularly report server application status.
+1. Regularly update the local cache from the remote interface.
+1. Regularly split files, delete temporary files.
+
+The framework provides a mechanism to make the development and maintenance of scheduled tasks more elegant.
+
+## Develope Scheduled Tasks
+
+All scheduled tasks are stored in directory `app/schedule`. Each file is an independent scheduled task that could configure the properties and the detail methods to be executed.
```

Comment:
how about change `are stored in` -> `should be placed in` ?

Reply:
agree